R/FilterStateLogical.R

```diff
@@ -1,35 +1,43 @@
+# LogicalFilterState ------
+
 #' @name LogicalFilterState
-#' @title `FilterState` object for logical variable
-#' @description Manages choosing a logical state
 #' @docType class
-#' @keywords internal
 #'
+#' @title `FilterState` object for logical variable
+#'
+#' @description Manages choosing a logical state.
 #'
 #' @examples
-#' filter_state <- teal.slice:::LogicalFilterState$new(
+#' # use non-exported function from teal.slice
+#' include_js_files <- getFromNamespace("include_js_files", "teal.slice")
+#' include_css_files <- getFromNamespace("include_css_files", "teal.slice")
+#' include_js_files <- getFromNamespace("include_js_files", "teal.slice")
```

Comment:
why include_js_files <- getFromNamespace("include_js_files", "teal.slice") is repeated twice? cc @kartikeyakirar